### PR TITLE
Add o-grid elements

### PR DESCRIPTION
**Pull request by keirog**
Zooming out before: | Zooming out after:
--- | ---
without-grid-container | with-grid-container

### CODE REVIEW

File: test/views/tour.spec.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const path = require('path');
+const tourLayoutHtml = fs.readFileSync(path.join(__dirname, '../../views') + '/tour.html', 'utf8');
+
+describe('Tour view', () => {
+	//Not an amazing test, but better than nothing…
+	it('contains `o-grid-container` somewhere', () => {
+		expect(tourLayoutHtml).to.contain('o-grid-container');
+	});
+
+	it('contains `id="site-content"` for n-ui’s “skip to content” link', () => {
```
Comment (laurieboyes) [inline test/views/tour.spec.js]: What's the 'skip to content' link?

Comment (keirog) [inline test/views/tour.spec.js]: All in the commit messages innit

Comment (keirog) [inline test/views/tour.spec.js]: Sorry that wasn’t helpful at all 😬 . There’s [a link in the header](https://github.com/Financial-Times/n-ui/blob/master/header/template.html#L2) for keyboard/screenreader users to skip to the main content of the page (skipping all the nav bits)

Comment (laurieboyes) [inline test/views/tour.spec.js]: It was very helpful, I didn't think to look at the commit messages. Cool 👍

**Pull request merged**